zeitschrift-fur-digitale-geisteswissenschaften.csl

```diff
@@ -0,0 +1,389 @@
+    </choose>
+  </macro>
+  <macro name="secondary-container-contributors">
+    <choose>
+      <if variable="container-author" type="chapter" match="all">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=", " delimiter-precedes-last="always"/>
+        </names>
+      </if>
+      <else-if type="article-journal article-magazine" match="any">
+        <names variable="editor">
+          <label form="verb-short" text-case="lowercase" suffix=". "/>
+          <name delimiter=" /" delimiter-precedes-last="always"/>
+        </names>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="secondary-collection-contributors">
+    <names variable="collection-editor">
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="translatedby">
+    <names variable="translator">
+      <label form="verb-short" text-case="lowercase" suffix=". "/>
+      <name delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <label form="verb-short" prefix=". " suffix=" "/>
+      <name delimiter=" / " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="title-long">
+    <group>
+      <choose>
+        <if type="book">
+          <text variable="title" form="long"/>
+          <text macro="editor"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="webpage">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="personal_communication">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="post-weblog">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="chapter">
+          <text variable="title" form="long"/>
+        </if>
+      </choose>
+      <choose>
+        <if type="article-journal">
+          <text variable="title" form="long" suffix="."/>
+        </if>
+      </choose>
+      <choose>
+        <if type="book">
+          <group>
+            <choose>
+              <if variable="volume">
+                <text prefix=". " term="volume" form="short" suffix=".&#160;"/>
+                <text variable="volume"/>
+              </if>
+            </choose>
+          </group>
+        </if>
+      </choose>
+      <text prefix=" / " macro="secondary-contributors"/>
+    </group>
+  </macro>
+  <macro name="title-short">
+    <group delimiter=". ">
+      <choose>
+        <if type="book">
+          <text variable="title" form="short"/>
+        </if>
+        <else>
+          <text variable="title" form="short"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <macro name="container">
+    <choose>
+      <if type="post-weblog">
+        <text prefix=" In: " variable="container-title"/>
```

Comment:
eleganter waere es sachen wie "In", "Bd." etc. mit `terms` zu loesen statt in die affixe zu setzen.